install/kubernetes/helm/istio/files/injection-template.yaml

```diff
@@ -333,4 +333,11 @@ dnsConfig:
     {{- range .Values.global.podDNSSearchNamespaces }}
     - {{ render . }}
     {{- end }}
-{{- end }}
\ No newline at end of file
+{{- end }}
+podRedirectAnnot:
```

Comment:
With merge of #13970 need to add `traffic.sidecar.istio.io/excludeOutboundPorts` as well.

Reply:
I can add here, but the change is missing on the CNI side so it won't change any behavior until that change goes in.

Reply:
Right... it will just help so when we do add the functionality to istio/cni this will already be in place.